templates/test-database-detection.yaml

```
@@ -0,0 +1,378 @@
+# Test Compose File for Database Detection Logic
+# This file contains various services to test the isDatabaseImage() function
+# and its contextual analysis capabilities
+# Service names indicate expected detection: "database-*" or "application-*"
+
+version: '3.8'
+
+services:
+  # =============================================================================
+  # ACTUAL DATABASES (should be detected as databases)
+  # =============================================================================
+  
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)